tests/commands/test_store_client.py

```diff
@@ -327,3 +354,50 @@ def test_client_clear_credentials():
     client.clear_credentials()
 
     assert mock_auth.clear_credentials.called_once_with()
+
+
+def test_client_errorparsing_complete():
+    """Produce a default message if response is not a json."""
+    content = json.dumps({"error-list": [{'message': 'error message', 'code': 'test-error'}]})
+    response = FakeResponse(content=content, status_code=404)
+    result = Client()._parse_store_error(response)
+    assert result == "error message [code: test-error]"
+
+
+def test_client_errorparsing_no_code():
+    """Produce a default message if response is not a json."""
+    content = json.dumps({"error-list": [{'message': 'error message', 'code': None}]})
+    response = FakeResponse(content=content, status_code=404)
+    result = Client()._parse_store_error(response)
+    assert result == "error message"
+
+
+def test_client_errorparsing_nojson():
+    """Produce a default message if response is not a json."""
+    response = FakeResponse(content='this is not a json', status_code=404)
+    result = Client()._parse_store_error(response)
+    assert result == "Failure working with the Store: [404] 'this is not a json'"
+
+
+def test_client_errorparsing_no_errors_inside():
+    """Produce a default message if response is not a json."""
```

Comment:
the comment is wrong here as are the next few tests

Reply:
Fixed, thanks